nixos/modules/services/misc/gitea.nix

```diff
@@ -517,7 +517,7 @@ in
     systemd.services.gitea = {
       description = "gitea";
       after = [ "network.target" ] ++ optional usePostgresql "postgresql.service" ++ optional useMysql "mysql.service";
-      requires = optional (cfg.database.createDatabase && usePostgresql) "postgresql.service" ++ optional (cfg.database.createDatabase && useMysql) "mysql.service";
+      requires = optional usePostgresql "postgresql.service" ++ optional useMysql "mysql.service";
```

Comment:
i hope someone who was never using `createDatabase` could chime in here: if you're using a remote postgresql instance, this change might be unwanted? i'm not totally sure how systemd `requires` interacts with services that aren't enabled.